test/fixtures/app/simple-app-custom-blueprint-test/my-app/package.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
This can be replaced by

```
{
  "name": "my-app",
  "version": "0.0.0",
  "description": "",
  "main": "index.js"
}
```

no need to store all this unused info.